graphs/adapter-jgrapht/src/main/kotlin/com/anaplan/engineering/azuki/graphs/adapter/jgrapht/action/JGraphTActionFactory.kt

```diff
@@ -40,3 +36,28 @@ val toJGraphTAction: (Action) -> JGraphTAction = {
     @Suppress("UNCHECKED_CAST")
     it as? JGraphTAction ?: throw IllegalArgumentException("Invalid action: $it")
 }
+
+private object JgraphtUndirectedGraphActionFactory : UndirectedGraphActionFactory {
```

Comment:
⛏️ The capitalisation of `JGraphT` should be consistent throughout all class names.
```suggestion
private object JGraphTUndirectedGraphActionFactory : UndirectedGraphActionFactory {
```